composer.json

```diff
@@ -14,26 +14,29 @@
     }
   },
   "require": {
-    "php": "~7.2",
+    "php": "^8.1",
     "ext-openssl": "*",
     "beberlei/assert": "^3",
-    "sensio/framework-extra-bundle": "^5.4",
-    "symfony/monolog-bundle": "^3.6.0",
-    "surfnet/stepup-saml-bundle": "^5.0",
-    "symfony/dependency-injection": "^4.4",
-    "symfony/framework-bundle": "^4.4"
+    "symfony/monolog-bundle": "^3.8",
+    "surfnet/stepup-saml-bundle": "^6.0",
+    "symfony/dependency-injection": "^5.4|^6.3",
```

Comment:
Do you wish to support 5.4?

Reply:
This was the choice when we started updating. Now that we do not support any SF5 it can be pinned to sf6 and php82